lf:
  ros__parameters:
    "bearing_std_dev": 1e-3
    "frame_id": "base_link"
    "least_sq_angle_thresh": 1e-4
    "least_sq_radius_thresh": 1e-4
    "max_line_gap": 0.4
    "max_range": 20.0
    "min_line_length": 0.5
    "min_line_points": 9
    "min_range": 0.4
    "min_split_dist": 0.05
    "outlier_dist": 0.05
    "publish_markers": true
    "range_std_dev": 0.02
    "scan_topic": "scan"